wandb/sdk/internal/sender.py

```diff
@@ -1507,7 +1514,7 @@ def _send_artifact(
         )
 
         self._job_builder._handle_server_artifact(res, artifact)
```

Comment:
Btw, this function is incorrect, it tries to guess the artifact version before it's committed. This is an existing issue though, not caused by your PR.

Reply:
:(